clients/admin-ui/src/pages/index.tsx

```diff
@@ -1,17 +1,61 @@
-import { Heading } from "@fidesui/react";
+import { Box, Button, Flex, Heading } from "@fidesui/react";
 import type { NextPage } from "next";
+import Head from "next/head";
+import NextLink from "next/link";
 import React from "react";
 
-import Layout from "~/features/common/Layout";
+import Header from "../features/common/Header";
+// import Setup from './setup';
+import { ArrowDownLineIcon } from "../features/common/Icon";
+
+const Home: NextPage<{ session: { username: string } }> = ({ session }) => (
+  <div>
+    <Head>
+      <title>Fides Admin UI</title>
+      <meta name="description" content="Generated from FidesUI template" />
+      <link rel="icon" href="/favicon.ico" />
+    </Head>
+
+    <Header username={session && session.username} />
 
-const Home: NextPage = () => (
-  <Layout title="Home">
     <main>
-      <Heading mb={8} fontSize="2xl" fontWeight="semibold">
-        Welcome home!
-      </Heading>
+      {/* NEED TO FLAG BEFORE RELEASE
+      Show Setup when the user is an admin and there are zero systems registered in the database */}
+      {/* <Setup /> */}
+
+      <Flex
+        borderBottom="1px"
+        borderTop="1px"
+        px={9}
+        py={1}
+        borderColor="gray.100"
+      >
+        <Button variant="ghost" mr={4} colorScheme="complimentary">
+          Subject Requests
+        </Button>
+        <Button variant="ghost" disabled mr={4}>
+          Datastore Connections
+        </Button>
+        <Button variant="ghost" disabled mr={4}>
+          User Management
+        </Button>
+        {/* This is a temporary link to the config wizard while it's still in progress */}
+        <NextLink href="/config-wizard" passHref>
+          <Button variant="ghost" disabled mr={4}>
```

Comment:
if we're adding a temporary link to make it easier to get to, we should un-disable it :)

Reply:
since we're planning on it being ready for the release, and we don't make claims that `main` is feature-complete, I'm ok leaving a link to it even if it isn't ready for prime time :)